kapacitor/client_test.go

```diff
@@ -11,9 +11,14 @@ import (
 )
 
 type MockKapa struct {
-	ResTask  client.Task
-	ResTasks []client.Task
-	Error    error
+	ResTask     client.Task
+	ResTasks    []client.Task
+	TaskError   error
+	UpdateError error
+	CreateError error
+	ListError   error
+	DeleteError error
```

Comment:
@timraymond we wanted to be more specific and complete with the types of errors we were expecting in the client test.